bottomless/src/bottomless_wal.rs

```diff
@@ -263,24 +161,4 @@ impl<T: Wal> Wal for BottomlessWal<T> {
 
         Ok(ret)
     }
-
-    fn exclusive_mode(&mut self, op: c_int) -> Result<()> {
```

Comment:
It's forwarded by default; notice how we are not implementing `Wal` but `WrapWal` now :) this is a convenient trait that helps compose WAL implementations. Upstream, I have stuff like:

```rust
        let wal_manager = Sqlite3WalManager::default()
            .wrap(TakeReaderWrapper(tmp.path().to_path_buf()))
            .wrap(ReplicationIndexInjectorWrapper);
```

so that it's super easy to only implement what you need for WAL implementation that wrap other WAL implementations